.circleci/config.yml

```diff
@@ -4430,6 +4430,31 @@ jobs:
           command: |
             ./release/scripts/vuln_check.sh || ./release/scripts/vuln_check.sh
 
+  scan-high-critical-vulnerabilities:
+    executor: custom
+    resource_class: small
+    steps:
+    - checkout
+    - setup_remote_docker
+    - run:
+        name: Install trivy
+        command: curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /tmp
+    - run:
+        name: Scan main image
+        command: /tmp/trivy image --exit-code 1 --severity HIGH,CRITICAL quay.io/stackrox-io/main:$(make tag)
```

Comment:
[no-op] Looks like there's `--format sarif`. GitHub likes Sarif. Although this workflow runs in CircleCI so I'm not sure how would pushing and viewing results through GitHub would be.

Reply:
Maybe that's the better direction to use it as github action and do nightly scan of latest image in quay and master branch code instead of having it on circleci where nobody will look

Reply:
https://github.com/stackrox/stackrox/pull/1672